php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -503,15 +500,6 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                     break;
 
                 case 'begintable':
-                    if ($addElements) {
```

Comment:
Why is this removed?

Reply:
that should have been done inanother PR butbasically it's because its an exact copy of `case 'endtable'`, I think it's a mistake anyways in the begin/end table logic. I will remove it from this PR and submit a PR specifically for it

Reply:
I don't think it is a mistake.. I think it's because of something about the smarty template for instruments using a table, and starting a new table needing to close that one to start a new table, while ending a table needs to end the table and re-start a table for the smarty template, but I can't remember the details.

Reply:
in that case their order of close first then open, or open then close should be reversed no ??

if not then you dont need to implement it twice in the switch which is what i did here... i just let it fall through

Reply:
@driusan restored